server/models/Conversation.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+
+const conversationSchema = new mongoose.Schema({
+  user1: {
+    type: mongoose.Types.ObjectId,
+    ref: "Profile",
+    required: [true, "A conversation must contain two profiles"],
+  },
+  user2: {
+    type: mongoose.Types.ObjectId,
+    ref: "Profile",
+    required: [true, "A conversation must contain two profiles"],
+  },
```

Comment:
It could make sense to turn these into an array for supporting groups. We could implement groups with this too (might involve a couple more controllers) or we could just adjust the design to prepare for it. I'll leave the decision up to you👍🏼

Reply:
I thought about turning this into an array called participants, but after reviewing the mocks. I found that idea unecesarry at this given time for this specific project.